src/app/organisms/settings/Settings.jsx

```diff
@@ -111,6 +118,46 @@ function AppearanceSection() {
   );
 }
 
+function NotificationsSection() {
+  const permission = usePermission('notifications', window.Notification?.permission);
+
+  const [, updateState] = useState({});
+
+  const renderOptions = () => {
+    if (window.Notification === undefined) {
+      return <Text className="set-notifications__not-supported">Not supported in this browser.</Text>;
+    }
+
+    if (permission === 'granted') {
+      return (
+        <Toggle
+          isActive={settings.showNotifications}
+          onToggle={() => { toggleNotifications(); updateState({}); }}
+        />
+      );
+    }
+
+    return (
+      <Button
+        variant="primary"
+        onClick={() => window.Notification.requestPermission()}
+      >
+        Request permission
+      </Button>
+    );
+  };
+
+  return (
+    <div className="set-notifications settings-content">
+      <SettingTile
+        title="Show notifications"
```

Comment:
Also shouldn’t ‘Show desktop notifications’ be more correct name for button?

Reply:
Element web calls them "desktop notifications". Element android calls them "notifications". Maybe calling them "System notifications" or "Native notifications" would make sense?

Reply:
I think Desktop notifications is much popular term and the one we should go with.